vividus-tests/src/main/resources/story/system/Kafka.story

```diff
@@ -14,13 +14,13 @@ Then `${consumed-messages[0]}` is equal to `${message}`
 
 
 Scenario: Wait until expected message appears in the Kafka topic
-When I start consuming messages from Kafka topics `${topic}`
-When I send data `{"key" : "failed"}` to Kafka topic `${topic}`
-When I send data `{"key" : "passed"}` to Kafka topic `${topic}`
+When I start consuming messages from `vividus` Kafka topics `${topic}`
+When I send data `{"key" : "failed"}` to `vividus` Kafka topic `${topic}`
+When I send data `{"key" : "passed"}` to `vividus` Kafka topic `${topic}`
 When I execute steps with delay `PT1S` at most 30 times while variable `messageCount` is = `0`:
 |step                                                                                                                               |
 |When I peek consumed Kafka messages to scenario variable `messages`                                                                |
```

Comment:
This step was deprecated and removed, it should be replaced with non-deprecated analogue